kornia/x/trainer.py

```diff
@@ -0,0 +1,158 @@
+import logging
+from typing import Callable, Dict
+
+import torch
+import torch.nn as nn
+from torch.utils.data import DataLoader
+
+# the accelerator library is a requirement for the Trainer
+# but it is optional for grousnd base user of kornia.
+try:
+    from accelerate import Accelerator
+except:
```

Comment:
```suggestion
except ImportError:
```
PEP8 talks about why bare except is a bad idea.
Also see https://realpython.com/the-most-diabolical-python-antipattern/